docs/pages/management/guides/awsoidc-integration.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: AWS OIDC Integration
+description: How to connect your AWS account with Teleport and provide access to AWS resources.
```

Comment:
I'm not sure about adding this to the "Run Teleport Agents" section, because we are not running a Teleport Agent.
We could also create a new section called "Cloud Access/AWS", but it would only have this integration for now.

> If this ends up being the preferred workflow to enroll Teleport resources in AWS

We don't support some of the most used (?) features, although that's currently being worked on.
I'm talking about
[EC2 Auto Discover using SSM](https://goteleport.com/docs/auto-discovery/servers/ec2-discovery/) https://github.com/gravitational/teleport/issues/41002
[AWS App Access](https://goteleport.com/docs/application-access/cloud-apis/aws-console/) https://github.com/gravitational/teleport/issues/30150
When we do, we might discuss if this should be the prefered way of enrolling aws resources

Reply:
Makes sense, thanks!